tests/helics/core/MessageTimerTests.cpp

```diff
@@ -0,0 +1,124 @@
+/*
+Copyright � 2017-2018,
+Battelle Memorial Institute; Lawrence Livermore National Security, LLC; Alliance for Sustainable Energy, LLC
+All rights reserved. See LICENSE file and DISCLAIMER for more details.
+*/
+#include "testFixtures.h"
+#include <boost/test/unit_test.hpp>
+
+#include "helics/core/MessageTimer.hpp"
+
+BOOST_AUTO_TEST_SUITE(messageTimer_tests)
+
+using namespace helics;
+
+BOOST_AUTO_TEST_CASE(basic_test)
+{
+    std::mutex mlock;
+    ActionMessage M;
+    auto cback = [&](ActionMessage &&m) {std::lock_guard<std::mutex> locker(mlock);
+    M = std::move(m); };
```

Comment:
This and lines 35-36 might be easier to read if the body of the lambda is on its own (indented) lines separate from the curly-braces.